scenario/LiquidationBotScenario.ts

```diff
@@ -122,6 +123,16 @@ for (let i = 0; i < MAX_ASSETS; i++) {
         ' == 500',
         // WBTC
         ' == 50'
+      ], 
+      'usdc.n': [
```

Comment:
Why are these values lower than the values for `arbitrum/usdc`? The liquidity for Arbitrum GMX, WETH and WBTC should be identical

Reply:
Let me tweak those number again. 

Yeah main reason to set those lower was due to the not having enough liquidity in the pool for USDC native now, and when liquidator bot try to borrow too much, it will fail. 

There is one scenario seems let liquidator bot to borrow as much as possible (with no max purchase setting), that one will fail if test user can borrow too big, and when liquidator bot attempts to borrow enough USDC to cover that debt.

Reply:
You should be able to borrow more if you switch to a `flashLoanPool` with greater liquidity. See: https://github.com/compound-finance/comet/pull/769#discussion_r1239026099

Reply:
Thanks after using the new pool suggested in the comment, I'm able to bump the volume back up without getting 'TF' error.

Reply:
Marking this as unresolved again since the liquidity in these tests is marked down again.

Reply:
Will leave it as unresolved. But as the time of testing today, the liquidity is enough again :P 
So reverted those number back up as the same as bridged version